server/views/pages/establishmentNotSupported.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "layout.njk" %}
+
+{% set pageHeaderTitle = "Selected establishment does not use this service" %}
+{% set pageTitle %}{{ applicationName }} - {{ pageHeaderTitle }}{% endset %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">{{ prisonName }} does not use this service</h1>
+
+      <p>Manage prison visits is a new service and only some establishments are using it.</p>
+      
+      <p>
+        If you have access to another establishment that uses this service,
+        click on the link in the DPS header to change your location.
```

Comment:
Argghh, who decided 'click' was a good word to use 😂 This has been bad accessibility practice since early 2000s.